### ISSUE TITLE
Possible future parameters/categories

### ISSUE THREAD

**Issue by bmkramer** [The-Academic-Observatory/query-auditing#4]
1. **version** (distinguish submitted, accepted and publshed version for green OA)
  * _available from Unpaywall_: yes (NB conservative detection)
  * _potential use case(s)_:  assess e.g. Plan S compliance
  
2. **date available oa** (for green oa only) 
  * _available from Unpaywall_: yes (for records from Aug 2020 forward))
  * _potential use case(s)_:  determine (and analyse) observed embargoes

3. **repository type** (e.g. PMC, repository of home institution)
  * _available from Unpaywall_: partly (depending on existence of specific url prefix for home repo)
  * _potential use case(s)_:  assess coverage of institutional repositories, assess effect of disciplinary repos vs. institutional repo's

**Comment by cameronneylon** [comment]
Note for date available oa - we could also get the date for the records previous to August 2020 by hitting the OAI-PMH entry for all the repository entries in Unpaywall data (probably on order 50 million API calls) if it was worth it.

**Comment by bmkramer** [comment]
I'm not sure though the OAI-PMH entry in all cases has the date the full-text was made available (e.g. in cases where the record was created earlier with the full-text under embargo)?

**Comment by cameronneylon** [comment]
That's true. Those were the dates that OurResearch used in their analysis but I'm not sure what that date field in Unpaywall data currently reports? Is it the first time Unpaywall crawler found full text or is it the OAI-PMH date field?

**Comment by bmkramer** [comment]
The information in the OAI-PMH date field at the time Unpaywall first found the full text :-) 

https://support.unpaywall.org/support/solutions/articles/44002063719
